docs/src/components/Document.js

```diff
@@ -10,7 +10,7 @@ const Document = ({ Body, children, Head, Html, siteData: { dev, versions } }) =
       <meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no' />
 
       <link rel='shortcut icon' type='image/x-icon' href='/logo.png' />
-      <link rel='stylesheet' href='/style.css' />
+      <link rel='stylesheet' href={`/style.css?${versions.suir}`} />
```

Comment:
We added changes to styles in #3215, but they can be cached because we don't manage these styles by Webpack (i.e. content hash or other hash).